runtime/src/bank.rs

```diff
@@ -3783,79 +3783,79 @@ impl Bank {
                 error_counters.account_not_found
             );
         }
         if 0 != error_counters.account_in_use {
-            inc_new_counter_info!(
+            inc_new_counter_debug!(
                 "bank-process_transactions-account_in_use",
                 error_counters.account_in_use
             );
         }
         if 0 != error_counters.account_loaded_twice {
-            inc_new_counter_info!(
+            inc_new_counter_debug!(
                 "bank-process_transactions-account_loaded_twice",
                 error_counters.account_loaded_twice
             );
         }
         if 0 != error_counters.blockhash_not_found {
-            inc_new_counter_info!(
+            inc_new_counter_debug!(
                 "bank-process_transactions-error-blockhash_not_found",
                 error_counters.blockhash_not_found
             );
         }
         if 0 != error_counters.blockhash_too_old {
-            inc_new_counter_info!(
+            inc_new_counter_debug!(
                 "bank-process_transactions-error-blockhash_too_old",
                 error_counters.blockhash_too_old
             );
         }
         if 0 != error_counters.invalid_account_index {
-            inc_new_counter_info!(
+            inc_new_counter_debug!(
                 "bank-process_transactions-error-invalid_account_index",
                 error_counters.invalid_account_index
             );
         }
         if 0 != error_counters.invalid_account_for_fee {
-            inc_new_counter_info!(
+            inc_new_counter_debug!(
                 "bank-process_transactions-error-invalid_account_for_fee",
                 error_counters.invalid_account_for_fee
             );
         }
         if 0 != error_counters.insufficient_funds {
-            inc_new_counter_info!(
+            inc_new_counter_debug!(
                 "bank-process_transactions-error-insufficient_funds",
                 error_counters.insufficient_funds
             );
         }
         if 0 != error_counters.instruction_error {
-            inc_new_counter_info!(
+            inc_new_counter_debug!(
                 "bank-process_transactions-error-instruction_error",
                 error_counters.instruction_error
             );
         }
         if 0 != error_counters.already_processed {
-            inc_new_counter_info!(
+            inc_new_counter_debug!(
                 "bank-process_transactions-error-already_processed",
                 error_counters.already_processed
             );
         }
         if 0 != error_counters.not_allowed_during_cluster_maintenance {
-            inc_new_counter_info!(
+            inc_new_counter_debug!(
                 "bank-process_transactions-error-cluster-maintenance",
                 error_counters.not_allowed_during_cluster_maintenance
             );
         }
         if 0 != error_counters.invalid_writable_account {
-            inc_new_counter_info!(
+            inc_new_counter_debug!(
```

Comment:
We actually use these errors quite a lot to debug transaction processing on the network. Could we aggregate these to report per slot like we do for `replay-slot-stats`: https://github.com/solana-labs/solana/blob/9052e41b3272cedcf63b063104fea16269a9749e/core/src/progress_map.rs#L41

Reply:
For this can we make it to report periodically via data point report?

Reply:
OK. Created a seperate issue to track this work: https://github.com/solana-labs/solana/issues/24456

Reply:
@carllin do you mind taking a look at this?